google/cloud/aiplatform/training_jobs.py

```diff
@@ -408,6 +474,38 @@ def _create_input_data_config(
                     key=predefined_split_column_name
                 )
 
+            # Create timestamp split spec
+            timestamp_split = None
+            if timestamp_split_column_name:
+                if dataset._gca_resource.metadata_schema_uri not in (
+                    schema.dataset.metadata.tabular,
+                    schema.dataset.metadata.time_series,
+                ):
+                    raise ValueError(
+                        "A timestamp split may only be used with a tabular or time series Dataset"
+                    )
+
+                timestamp_split = gca_training_pipeline.TimestampSplit(
+                    training_fraction=training_fraction_split,
+                    validation_fraction=validation_fraction_split,
+                    test_fraction=test_fraction_split,
+                    key=timestamp_split_column_name,
+                )
+
+            fraction_split = None
+            if all(
```

Comment:
I think here we should be checking if we received any of the fraction split arguments, and not the timestamp split, and, if so, create the FractionSplit proto and default any missing values to there defaults in previous versions.

Then below this there should be a block like this:

```python
n_split_configs = len([split for split in [fraction_split, filter_split, timestamp_split, predefined_split] if split is not None])
if  n_split_configs > 1:
  raise ValueError("Can only specify one of:"
     "1. training_fraction_split, validation_fraction_split, testing_fraction_split"
     "2. training_fraction_split, validation_fraction_split, testing_fraction_split, timestamp_split,"
     "3. ....")
     
if n_split_configs = 0:
  # create default 0.8, 0.1, 0.1
```